src/Modal/index.jsx

```diff
@@ -119,7 +171,7 @@ class Modal extends React.Component {
         <div className={styles['modal-dialog']}>
           <div className={styles['modal-content']}>
             <div className={styles['modal-header']}>
-              <h5 className={styles['modal-title']} id={this.headerId}>{this.props.title}</h5>
+              <h2 className={styles['modal-title']} id={this.headerId}>{this.props.title}</h2>
```

Comment:
Will changing this negatively impact any of our current modals?

Reply:
It should not - the change here to `h2` comes from a quick discussion with Ari & Mark S. noting that it would be expected to be that type.

Reply:
👍  sweet, just wanted to check